xclim/core/units.py

```diff
@@ -373,9 +370,7 @@ def convert_units_to(  # noqa: C901
     raise NotImplementedError(f"Source of type `{type(source)}` is not supported.")
 
 
-def cf_conversion(
-    standard_name: str, conversion: str, direction: Literal["to", "from"]
-) -> str | None:
+def cf_conversion(standard_name: str, conversion: str, direction: str) -> str | None:
```

Comment:
I'm ok with this. But to understand, why was the stricter annotation removed ?

Reply:
You know, I think that might've been a false-positive from `pylint`. The Literal method works fine in my IDE. I'll undo this change.